libgraph/include/katana/PropertyGraph.h

```diff
@@ -239,11 +243,11 @@ class KATANA_EXPORT PropertyGraph {
       EntityTypeManager&& edge_type_manager) noexcept
       : rdg_(std::move(rdg)),
         file_(std::move(rdg_file)),
-        node_entity_type_manager_(std::move(node_type_manager)),
-        edge_entity_type_manager_(std::move(edge_type_manager)),
         node_entity_type_ids_(std::move(node_entity_type_ids)),
         edge_entity_type_ids_(std::move(edge_entity_type_ids)),
-        pg_view_cache_(std::move(topo)) {
+        pg_view_cache_(std::move(topo)),
```

Comment:
Is this a style-only change, or accidentally included, or am I missing something?

Reply:
those member variables got moved below in the declaration to make them protected, and the initialization needs to match the declaration